### ISSUE TITLE
"Play" button doesn't work

### ISSUE THREAD

**Issue by mcmirande** [CDrummond/cantata#1327]
Hello, at first thanks for Cantata!  It's my preferred music player in Archlinux

I write here because the "play" button doesn't work since I tried to install "Soundwire" last night.  Cantata works to manage my music, edit tags and covers... but I can't reproduce music.

When I execute it from console, the following message appear:

inotify_add_watch("/home/marcos") failed: "No space left on device"
inotify_add_watch("/") failed: "No space left on device"

The MPD server is working (Sonata works), but I don't know how to search where is the problem with Cantata.  I'm sorry if there's a trivial solution that I don't know...

Thanks for any help!
Marcos.

**Comment by CDrummond** [comment]
Sorry - but the play button **does** work (just confirmed). You have problems with your system, not much I can do.

**Comment by mcmirande** [comment]
Yes, :) 

I'm sorry.  I know the problem is with my system, but I haven't a clue about how to solve it...

Thanks, Marcos

**Comment by trmdi** [comment]
@mcmirande 

Do you have the latest version of Cantata and mpd ?
Anyways, you can get some debug info with this command: `$ pkill mpd ; cantata -d all`

**Comment by mcmirande** [comment]
Thanks to all.  I found what was wrong

I don't know why but my music directory changed with some of the things I did.  As Cantata (until I understand) saves a copy of the music directory, it appeared as it were reading it... but not...

I fixed that and got Cantata working again.

One more question, to fetch Spotify artworks is necessary to have a premium account, or it wasn't implemented yet?

Thanks again!  Marcos
